src/microbots/llm/openai_api.py

```diff
@@ -38,12 +39,12 @@ def ask(self, message) -> llmAskResponse:
         self.messages.append({"role": "user", "content": message})
         return_value = {}
         while self._validate_llm_response(return_value) is False:
-            response = self.ai_client.responses.create(
+            response = self.ai_client.chat.completions.create(
```

Comment:
Our intention to use `responses` api over `chat_completion` api is to depending on the model itself to maintain the context using `store=true`. We're yet to invest and implement that feature in detail. So, I would recommend you to stick with `responses` api itself as much as possible.

If you still wish to stick with `chat_completion`, please create a new class `OpenAIChatCompletionAPI` class and implement your changes. When introducing a new such class, please create one `Abstract class` and implement it in both of the API classes (Later we'll move to Factory design patter or Pydantic based models).

We'll have provision to include appropriate API class in `CustomBots`. So, the user can choose their required API class based on the model they use.